constants/routes.js

```diff
@@ -0,0 +1,23 @@
+import ICONS from "./icons"
+
+const ROUTES = {
+    HOME: '/',
+    navbarPages: [
```

Comment:
the navbarPages array can either be defined in the file where the component that would use it is rendered or you can create a separate variable within this file `routes.js` like

```
const PAGES = [...]

export { PAGES } ;
```